Dockerfile

```
@@ -0,0 +1,17 @@
+FROM maven:3-eclipse-temurin-23 AS builder
+COPY ./ /app
+COPY pom.xml /app
+RUN mvn --no-transfer-progress --file /app/pom.xml clean package -DskipTests
+WORKDIR application
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)